tests/test_spherePoint.py

```diff
@@ -188,38 +207,43 @@ def testGetLatitudeValue(self):
-            self.assertIsInstance(point.getLatitude(), afwGeom.Angle)
-            # Behavior for non-finite points is undefined; depends on internal
-            # data representation
-            if point.isFinite():
-                self.assertGreaterEqual(point.getLatitude().asDegrees(), -90.0)
-                self.assertLessEqual(point.getLatitude().asDegrees(), 90.0)
-                self.assertAnglesAlmostEqual(lat, point.getLatitude())
+            for point in (
+                SpherePoint(lon, lat),
+                SpherePoint(lon.asDegrees(), lat.asDegrees(), degrees),
+                SpherePoint(lon.asRadians(), lat.asRadians(), radians),
+            ):
+                self.assertIsInstance(point.getLatitude(), afwGeom.Angle)
+                # Behavior for non-finite points is undefined; depends on internal
+                # data representation
+                if point.isFinite():
+                    self.assertGreaterEqual(point.getLatitude().asDegrees(), -90.0)
+                    self.assertLessEqual(point.getLatitude().asDegrees(), 90.0)
+                    self.assertAnglesAlmostEqual(lat, point.getLatitude())
 
     def testGetVectorValue(self):
         """Test if getVector() returns the expected value.
 
         The test includes conformance to vector-angle conventions.
         """
-        pointList = [
-            ((0.0, 0.0), afwGeom.Point3D(1.0, 0.0, 0.0)),
-            ((90.0, 0.0), afwGeom.Point3D(0.0, 1.0, 0.0)),
-            ((0.0, 90.0), afwGeom.Point3D(0.0, 0.0, 1.0)),
-        ]
-
-        for lonLat, vector in pointList:
-            # Convert to Point3D.
-            point = SpherePoint(lonLat[0]*degrees, lonLat[1]*degrees)
-            newVector = point.getVector()
-            self.assertIsInstance(newVector, afwGeom.Point3D)
-            for oldElement, newElement in zip(vector, newVector):
-                self.assertAlmostEqual(oldElement, newElement)
-
-            # Convert back to spherical.
-            newLon, newLat = SpherePoint(newVector)
-            self.assertAlmostEqual(newLon.asDegrees(), lonLat[0])
-            self.assertAlmostEqual(newLat.asDegrees(), lonLat[1])
+        for lon, lat, vector in [
+            (0.0*degrees, 0.0*degrees, afwGeom.Point3D(1.0, 0.0, 0.0)),
+            (90.0*degrees, 0.0*degrees, afwGeom.Point3D(0.0, 1.0, 0.0)),
+            (0.0*degrees, 90.0*degrees, afwGeom.Point3D(0.0, 0.0, 1.0)),
+        ]:
+            for point in (
+                SpherePoint(lon, lat),
+                SpherePoint(lon.asDegrees(), lat.asDegrees(), degrees),
+                SpherePoint(lon.asRadians(), lat.asRadians(), radians),
+            ):
+                newVector = point.getVector()
+                self.assertIsInstance(newVector, afwGeom.Point3D)
+                for oldElement, newElement in zip(vector, newVector):
+                    self.assertAlmostEqual(oldElement, newElement)
+
+                # Convert back to spherical.
```

Comment:
If you're removing the "Convert to Point3D" comment, you might remove this one as well

Reply:
Sure. I thought I had reverted the other comment to its original place but maybe not. I'll remove both of them as I don't think they add much.